readme.md

```diff
@@ -352,6 +352,7 @@
 - [Static Analysis & Code Quality](https://github.com/analysis-tools-dev/static-analysis#readme)
 - [Information Retrieval](https://github.com/harpribot/awesome-information-retrieval#readme) - Learn to develop your own search engine.
 - [Quantum Computing](https://github.com/desireevl/awesome-quantum-computing#readme) - Computing which utilizes quantum mechanics and qubits on quantum computers.
+- [Theoretical Computer Science](https://github.com/mostafatouny/awesome-theoretical-computer-science/) - The interplay of computer science and pure mathematics, distinguished by its emphasis on mathematical rigour and technique.
```

Comment:
You're missing `#readme`.

Reply:
![_img20216262028](https://user-images.githubusercontent.com/82480625/124761965-a4224400-df32-11eb-9b31-eb7e939bd078.png)
@joramtech